src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  padding-top: 16px;
+  align-items: end;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    &:nth-child(-n+4) {
```

Comment:
The star quantity should depend on the class with the modifier. You can use styles from the previous task